packages/components/src/drop-zone/index.js

```diff
@@ -95,7 +139,7 @@ export default function DropZoneComponent( {
 
 	return (
 		<div ref={ ref } className={ classes }>
-			{ children }
+			<AnimatePresence>{ children }</AnimatePresence>
```

Comment:
Maybe this animate presence should move to line 109 and only be added when there's an actual animation?